spec/models/user_spec.rb

```diff
@@ -1,10 +1,26 @@
 require 'spec_helper'
 
 describe User do
+  around(:each)do
+   |example| ActionMailer::Base.deliveries.clear
```

Comment:
this `around` block is also entirely unnecessary. `spec_helper` already does this.